src/java/org/apache/cassandra/service/accord/AccordService.java

```diff
@@ -973,6 +1096,30 @@ private static CommandStoreTxnBlockedGraph.TxnState populate(CommandStoreTxnBloc
         return cmdTxnState.build();
     }
 
+    @Nullable
+    @Override
+    public Long minEpoch(Collection<TokenRange> ranges)
+    {
+        return node.topology().minEpoch();
+    }
+
+    @Override
+    public void tryMarkRemoved(Topology topology, Id target)
+    {
+        if (node.commandStores().count() == 0) return; // when starting up stores can be empty, so ignore
+        Ranges ranges = topology.rangesForNode(target);
+        if (ranges.isEmpty()) return;
+        tryMarkRemoved(ranges, 0).begin(node().agent());
```

Comment:
we don't need to wait on these sync points... we just need to schedule them and hope for the best... its kinda "yolo" semantics until we have something working that is dealing with `ExclusiveSyncPoints`